demo/react-native/ios/Podfile

```diff
@@ -8,3 +8,11 @@ target 'CheetahDemo' do
   pod 'Cheetah-iOS', '~> 2.0.0'
```

Comment:
I don't think this is supposed to be here - it should get Cheetah-iOS from the react native SDK